proxycore/connpool.go

```diff
@@ -209,6 +211,7 @@ func (p *connPool) stayConnected(idx int) {
 				done = true
 				_ = conn.Close()
 			case <-conn.IsClosed():
+				p.logger.Warn("pool closed", zap.Stringer("host", p.config.Endpoint))
```

Comment:
I'm not sure if `Warn` is the best log level here. In Astra coordinator topology changes are somewhat common (tenant allocation/deallocation, upgrades) so it might "scare" users without a good reason when they eventually see these warnings.

Reply:
good call.